views/layout-designer-page.twig

```diff
@@ -74,12 +74,12 @@
                     edit: "{% trans "Edit layout regions" %}",
                     addRegion: "{% trans "Add" %}",
                     addRegionDesc: "{% trans "Add a new region" %}",
-                    deleteRegion: "{% trans "Delete" %}",
+                    deleteRegion: "{% trans "Delete region" %}",
```

Comment:
I think we ought to have a capital R for region where and on the prior lines. This would be consistent with our use of capital letters for naming "Xibo objects".